src/promptflow-devkit/promptflow/_proxy/_python_inspector_proxy.py

```diff
@@ -35,7 +36,10 @@ def get_entry_meta(
         **kwargs,
     ) -> Dict[str, Any]:
         timeout = kwargs.get("timeout", FLOW_META_JSON_GEN_TIMEOUT)
-        load_in_subprocess = kwargs.get("load_in_subprocess", True)
+        load_in_subprocess = (
+            kwargs.get("load_in_subprocess", True)
```

Comment:
why do we need to have two ways to configure here? 
are both of them needed to be configured? is this by design?

Reply:
`load_in_subprocess` exists before but it is not being used anywhere at the moment. Removing it.